src/components/ebay-checkbox/test/test.server.js

```diff
@@ -0,0 +1,36 @@
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const rootTag = `span.checkbox`;
+const inputTag = `input.checkbox__control`;
+const iconTag = `span.checkbox__icon`;
```

Comment:
Let's call these selectors for clarity (`rootSelector` `inputSelector` `iconSelector`).